pkg/service/controller/servingcert/secret_creating_controller.go

```diff
@@ -207,37 +207,40 @@ func (sc *ServiceServingCertController) work() bool {
 // syncService will sync the service with the given key.
 // This function is not meant to be invoked concurrently with the same key.
 func (sc *ServiceServingCertController) syncService(key string) error {
-	obj, exists, err := sc.serviceCache.GetByKey(key)
+	namespace, name, err := cache.SplitMetaNamespaceKey(key)
 	if err != nil {
-		glog.V(4).Infof("Unable to retrieve service %v from store: %v", key, err)
 		return err
 	}
-	if !exists {
+	service, err := sc.serviceLister.Services(namespace).Get(name)
```

Comment:
call this one `sharedService` so I don't have to look to make sure you got all the old `service` calls?